lib/tools/node-modules/node-modules-builder.ts

```diff
@@ -133,27 +133,156 @@ export class NodeModulesBuilder implements INodeModulesBuilder {
 				// Force copying if the destination doesn't exist.
 				lastModifiedTime = null;
 			}
-			let nodeModules = this.getChangedNodeModules(absoluteOutputPath, platform, lastModifiedTime).wait();
 
-			const resolver = new NpmDependencyResolver(this.$projectData.projectDir);
-			const resolvedDependencies = resolver.resolveDependencies(_.keys(nodeModules), platform);
+			let productionDependencies = this.getProductionDependencies(this.$projectData.projectDir);
+
+			// console.log(productionDependencies);
+
+			// TODO: Pip3r4o - result is not used currently
+			// let nodeModules = this.getChangedNodeModules(absoluteOutputPath, platform, lastModifiedTime).wait();
 
 			if (!this.$options.bundle) {
 				const tnsModulesCopy = this.$injector.resolve(TnsModulesCopy, {
 					outputRoot: absoluteOutputPath
 				});
-				tnsModulesCopy.copyModules(resolvedDependencies, platform);
+				tnsModulesCopy.copyModules(productionDependencies, platform);
 			} else {
 				this.cleanNodeModules(absoluteOutputPath, platform);
 			}
 
 			const npmPluginPrepare = this.$injector.resolve(NpmPluginPrepare, {});
-			npmPluginPrepare.preparePlugins(resolvedDependencies, platform);
+			npmPluginPrepare.preparePlugins(productionDependencies, platform);
 		}).future<void>()();
 	}
 
+	public getProductionDependencies(projectPath: string) {
+		let deps: any = [];
+		let seen: any = {};
+
+		let pJson = path.join(projectPath, "package.json");
+		let nodeModulesDir = path.join(projectPath, "node_modules");
+
+		let content = require(pJson);
+
+		Object.keys(content.dependencies).forEach((key) => {
```

Comment:
if there's no dependencies section in the package.json, this code will fail. I would use lodash's `_.each`:
```TypeScript
_.keys(content.dependencies)
   .forEach(key => { 
    ....
    });
```